python/vineyard/cli.py

```diff
@@ -29,10 +29,23 @@
 
 def vineyard_argument_parser():
     """Utility to create a command line Argument Parser."""
-    parser = ArgumentParser(prog='vineyard-ctl',
-                            usage='%(prog)s [options]',
-                            description='vineyard-ctl: A command line tool for vineyard',
-                            allow_abbrev=False)
+    parser = argparse.ArgumentParser(
+        prog='vineyard-ctl',
+        usage='%(prog)s [options]',
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description='vineyard-ctl: A command line tool for vineyard',
+        allow_abbrev=False,
+        epilog=('Some examples on how to use vineyard-ctl:\n\n1. Connect to a vineyard server\n>>> vineyard-ctl ' +
+                '--ipc_socket /var/run/vineyard.sock\n\n2. List vineyard objects\n' +
+                '>>> vineyard-ctl ls --limit 6\n\n3. Query a vineyard object\n>>> ' +
+                'vineyard-ctl query --object_id 00002ec13bc81226\n\n4. Print first n lines' +
+                ' of a vineyard object\n>>> vineyard-ctl head --object_id 00002ec13bc81226' +
+                ' --limit 3\n\n5. Copy a vineyard object\n>>> vineyard-ctl copy --object_id' +
+                ' 00002ec13bc81226 --shallow\n\n6. Delete a vineyard object\n>>> vineyard-ctl' +
+                ' del --object_id 00002ec13bc81226\n\n7. Get the status of connected vineyard' +
+                ' server\n>>> vineyard-ctl stat\n\n8. Put a python value to vineyard\n>>> ' +
+                'vineyard-ctl put --file example_csv_file.csv\n\n9. Edit configuration file\n' +
+                '>>> vineyard-ctl config --ipc_socket_value /var/run/vineyard.sock'))
```

Comment:
Could you please put those lines as a miltiple-line string like

```python
examples = """
Some examples on how to use vineyard-ctl:

1. Connect to a vineyard server

    >>> vineyard-ctl --ipc_socket /var/run/vineyard.sock

2. List vineyard objects

    ....
"""
```

and use it like

```python
epilog = example,
```

Reply:
Yes, multiple-line string could preserve the original formats and spaces, make your life easier XD

Reply:
I just updated the examples to be a multi-line string. :)